client/extensions/wp-super-cache/style.scss

```diff
@@ -151,3 +151,23 @@
 .wp-super-cache__cdn-fieldsets {
 	margin-left: 40px;
 }
+
+.wp-super-cache__main {
```

Comment:
The Calypso guidelines actually discourage nesting - https://wpcalypso.wordpress.com/devdocs/docs/coding-guidelines/css.md#nesting